packages/saudi-arabia/README.md

```diff
@@ -0,0 +1,39 @@
+# sa Saudi Arabia
+
```

Comment:
Could you [add this country in the main `README.md` file](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#documentation) please?